src/app/app.component.ts

```diff
@@ -56,19 +58,33 @@ export class AppComponent implements OnInit {
         canvas.width = img.width;
         canvas.height = img.height;
         ctx.drawImage(img, 0, 0);
-        console.log(canvas.width, img.width, 'width');
-        console.log(canvas.height, img.height, 'height');
-        this.imgSize = canvas.height + ' x ' + canvas.width;
       };
       img.src = e.target['result'];
     };
+    this.imgName = this.buildings[this.index]['name'];
     reader.readAsDataURL(this.buildings[this.index]);
   }
 
   public resetImage() {
     this.processCanvas();
   }
 
+  public saveToCsv() {
+    let csvContent = 'data:text/csv;charset=utf-8,';
```

Comment:
Am I missing something or doesn't the csv include a header? It seems to me you're just writing the values. 

(If this is intended behaviour, fine for me, just wanted to point it out)

Reply:
yes its just the values. I wasn't sure if we need headers. If so, then let me know & i will add them

Reply:
I'd prefer it so we don't have to assume (hardcode) it when parsing the files.

Reply:
Okay then how would you call them?

Img - Upper-left X - Upper-left Y - Lower-right X - Lower-right Y ?

Reply:
Yes.

Reply:
done